roles/lib_openshift/src/doc/ca_server_cert

```diff
@@ -0,0 +1,96 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oc_adm_ca_server_cert
+short_description: Module to run openshift oc adm ca create-server-cert
+description:
+  - Wrapper around the openshift `oc adm ca create-server-cert` command.
+options:
+  state:
+    description:
+    - Present is the only supported state.  The state present means that `oc adm ca` will generate a certificate
+    - and verify if the hostnames and the ClusterIP exists in the certificate.
+    - When create-server-cert is desired then the following parameters are passed.
+    - ['cert', 'key', 'signer_cert', 'signer_key', 'signer_serial']
+    required: false
+    default: present
+    choices: 
+    - present
+    aliases: []
+  kubeconfig:
+    description:
+    - The path for the kubeconfig file to use for authentication
+    required: false
+    default: /etc/origin/master/admin.kubeconfig
+    aliases: []
+  debug:
+    description:
+    - Turn on debug output.
+    required: false
+    default: False
+    aliases: []
+  cert:
+    description:
+    - The certificate file. Choose a name that indicates what the service is.
+    required: false
+    default: None
+    aliases: []
+  key:
+    description:
+    - The key file. Choose a name that indicates what the service is.
+    required: false
+    default: None
+    aliases: []
+  force:
+    description:
+    - Force updating of the existing cert and key files
+    required: false
+    default: False
+    aliases: []
+  signer_cert:
+    description:
+    - The signer certificate file.
+    required: false
+    default: /etc/origin/master/ca.crt
+    aliases: []
+  signer_key:
+    description:
+    - The signer key file.
+    required: false
+    default: /etc/origin/master/ca.key
+    aliases: []
+  signer_serial:
+    description:
+    - The signer serial file.
+    required: false
+    default: /etc/origin/master/ca.serial.txt
+    aliases: []
+  hostnames:
+    description:
+    - Every hostname or IP that server certs should be valid for (comma-delimited list)
```

Comment:
If I read the code correctly, the ansible module takes a real list, it's only comma-delimited string internally in CAServerCertConfig